src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -229,6 +241,14 @@ public void setTokenScannerSymbols(TokenScannerSymbols tokenScannerSymbols) {
     this.tokenScannerSymbols = tokenScannerSymbols;
   }
 
+  public Set<Class> getAllowedHostClasses() {
```

Comment:
maybe it's worth wrapping these in a `MethodSecurityConfig` or `ClassSecurityConfig` class or something? The names are not terribly easy to understand. 

Then you could just call this allowedClasses.